### PR TITLE
added thank you message in traditional Chinese & added photos and info about mentors

### PR DESCRIPTION
**Pull request by izzylys**
I used Google translate, so I hope the translation is accurate since I do not speak Chinese myself.

感謝您的貢獻 !

### CODE REVIEW

File: _data/mentors.json [hunk 1]
```diff
@@ -108,7 +108,14 @@
...
 {
-    "name": "Sandareka Wickramanayake"
+    "social": {
+        "github": "sandareka",
+        "link": "http://sandaw89.blogspot.com/",
+        "twitter": "wsanda89",
+        "facebook": "sandareka.wickramanayake"
+    },
+    "imgName": "Wickramanayake.jpg",
+	"name": "Sandareka Wickramanayake"
```
Comment (roonyh) [inline _data/mentors.json]: Use spaces here (and other applicable locations) for indentation. Please don't use tabs.

**Comment by roonyh** [comment]
Thanks much! I am not sure if Google translate is 100% correct. Maybe someone who speaks it could help us confirm? @fossasia/fossasia-gci @fossasia/fossasia-gci-students ?

Please make the bigger images reduced to 300px*300px. And fix the small issues with indentation.

**Comment by izzylys** [comment]
Hello Aruna!

Thank you for your feedback!  I just fixed the errors you mentioned.  The images have been resized and all tabbed indentations have been replaced by spaces.  Thanks!

**Comment by wongalvis** [comment]
@RoonyH @izzylys The translation is good. :)

**Comment by izzylys** [comment]
@pwmvx1289 Thank you for confirming!

**Pull request merged**